### PR TITLE
Reset the rotation of items

### LINKED ISSUE #10507: [BUG] No way to reset rotation of items.
### There is no other issue reporting the same problem.

- [X] I have searched the existing issues

### Game Version

24082318

### I was

Client

### OS

Windows 8, 10, 11

### What happened?

When you throw an item the rotation of it can become off. Previously picking up a item and dropping it would reset it back to how it's supposed to be rotated. But now that does not work.

### What should have happened instead?

Picking up an item should reset the rotation it had once you drop it again, or perhaps just a right click open to reset a item's rotation.

### Steps to reproduce the issue

1. Throw an item
2. Pick it up
3. Drop it

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MaxIsJoe**
LISTEN HERE, LIBERAL.
SCRIPTS ARE MEANT TO HAVE A SINGLE RESPONSIBILITY.
BACK IN MY DAY, A MOVEMENT SCRIPT ONLY HANDLED MOVEMENT; NOT HOW WE SHOVE THE BRITISH INTO LOCKERS.
I'M BRINGING BACK SINGLE RESPONSIBILITY, AND THERE'S NOTHING YOU CAN DO ABOUT THIS.
### **AND YOU BETTER PRAY TO GOD THAT I DON'T CREATE ANOTHER PARTIAL CLASS SCRIPT FOR THE LIFECYCLES AND THE NEWTONIAN MOVEMENT.**

oh yeah also closes #10507 since people have been wanting to reset the rotation of items for a while.

# Changelog

CL: [New] Added the ability to reset the rotation of items via a right click command.

### CODE REVIEW

**Comment by Bod9001** [comment]
Fix merge conflicts

**Comment by MaxIsJoe** [comment]
fixed

File: UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.cs [hunk 1]
```diff
@@ -1192,12 +1192,13 @@ private void AirTimeChecks()
 				var floating = IsFloating();
 				if (floating == false)
 				{
-					AppliedFriction(DEFAULT_SLIDE_FRICTION);
+					AppliedFriction(DEFAULT_Friction);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.cs]: oops this is my bad should be DEFAULT_SLIDE_FRICTION xD

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.cs]: done

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**